qa_eval/evaluation.py

```diff
@@ -24,48 +26,77 @@ def evaluate_steps(
 
 def run_evaluation(
         gsc_templates: list[dict],
```

Comment:
We can rename `gsc_templates`  to` qa_dataset` or something like that. As far as I remember we would like to use the term Q&A dataset instead of GSC